Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM azul/zulu-openjdk:21-latest AS build
 ENV GRADLE_OPTS="-Dorg.gradle.daemon=false -Dkotlin.incremental=false"
 WORKDIR /app
 
-COPY gradlew settings.gradle .editorconfig ./
+COPY gradlew settings.gradle .editorconfig gradle.propertie[s] ./
```

Comment:
Why the brackets?

Reply:
globbing in a Dockerfile COPY allows us to not fail the copy when the file doesn't exist.

the bracket match only one file named 'gradle.properties'

credit: https://stackoverflow.com/questions/31528384/conditional-copy-add-in-dockerfile

Reply:
Oh cool. Thanks for explaining.